### PR TITLE
PYIC-7135 Resolve parsing issue for report output result. 

### PR DESCRIPTION
**Pull request by rpayal**
## Proposed changes
Resolve parsing issue for report output .

### What changed
added - objectMapper.disable(SerializationFeature.FAIL_ON_EMPTY_BEANS);

### Issue tracking

- [PYIC-7135](https://govukverify.atlassian.net/browse/PYIC-7135)

### Environment variables or secrets

- [ ] No environment variables or secrets were added or changed

- [ ] Documented in the [README](./blob/main/README.md)
- [ ] Added to deployment repository
- [ ] Added to local startup repository

### Other considerations

- [ ] Update [README](./blob/main/README.md) with any new instructions or tasks

[PYIC-7135]: https://govukverify.atlassian.net/browse/PYIC-7135?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/ReportUserIdentityHandler.java [hunk 1]
```diff
@@ -45,7 +46,7 @@ public class ReportUserIdentityHandler implements RequestStreamHandler {
     public static final int STOP_TIME_IN_MILLISECONDS_BEFORE_LAMBDA_TIMEOUT = 60000;
     public static final String ATTR_NAME_USER_ID = "userId";
     private static final Logger LOGGER = LogManager.getLogger();
-    private final ObjectMapper objectMapper;
+    private static final ObjectMapper objectMapper = new ObjectMapper();
```
Comment (Joe-Edwards-GDS) [inline lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/ReportUserIdentityHandler.java]: (if it's static it should really be `OBJECT_MAPPER`)

File: libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/DynamoDbHelper.java [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+                        AttributeValue::n,
+                        AttributeValue::bool,
+                        AttributeValue::ns,
+                        AttributeValue::ss,
+                        AttributeValue::b,
+                        AttributeValue::bs)
+                .map(f -> f.apply(av))
+                .filter(Objects::nonNull)
+                .findFirst()
+                .orElseThrow();
```
Comment (Joe-Edwards-GDS) [inline libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/DynamoDbHelper.java:94]: Is this copied from somewhere? It's an interesting construction...

File: libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/DynamoDbHelper.java
Comment (Joe-Edwards-GDS) [inline libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/DynamoDbHelper.java:1]: This is neat, although seems to be a bit more detailed than we actually need - do we have anything other than `String` fields in our last evaluated keys? We could potentially cut out all the handling of other types, and throw `UnsupportedOperationException` to simplify.

But you've written it now...

Comment (Joe-Edwards-GDS) [inline libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/DynamoDbHelper.java:1]: (An even neater solution would be to set this up as a custom Jackson serializer/deserializer, but we don't expect this to be widely used)

**Review by Joe-Edwards-GDS — DISMISSED** [review]
A couple of comments if you make any more changes, but ok to merge

**Review by Joe-Edwards-GDS — APPROVED** [review]

File: lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/ReportUserIdentityHandler.java (hunk 1 above)
Comment (rpayal) [reply lambdas/report-user-identity/src/main/java/uk/gov/di/ipv/core/reportuseridentity/ReportUserIdentityHandler.java]: Done

File: libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/DynamoDbHelper.java (hunk 2 above)
Comment (rpayal) [reply libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/DynamoDbHelper.java:94]: yeah google.

**Pull request merged**